src/engine/Graphics/GraphicsSystem.ts

```diff
@@ -265,7 +265,7 @@ export class GraphicsSystem extends System {
       }
 
       if (transform) {
-        this._graphicsContext.z = transform.z;
+        this._graphicsContext.z = transform.globalZ;
         this._graphicsContext.translate(tx.pos.x, tx.pos.y);
```

Comment:
@eonarheim unsure if this is the right approach here. the other values aren't using the global here, so this leads me to think `translate` should maybe be updated to take into account the z?

Reply:
I think that global z is right, the graphics context uses the z to sort so this seems right

Reply:
gotcha. well, it works like this, so i'm fine to leave it like so if you are.